zingolib/src/commands.rs

```diff
@@ -824,16 +820,69 @@ impl Command for SendCommand {
                 )
             }
         };
-        for send in &send_inputs {
-            let address = &send.0;
-            let memo = &send.2;
-            if memo.is_some() && is_transparent_address(address, &lightclient.config.chain) {
+        if let Err(e) = utils::check_memo_compatibility(&send_inputs, &lightclient.config().chain) {
```

Comment:
Please helperize wet code.

Reply:
I did helperise into check_memo_compatibility as requested ... this error handling looks dry to me. .. can you provide a suggestion?